CGMBLEKit/TransmitterManager.swift

```diff
@@ -210,6 +210,7 @@ public class TransmitterManager: TransmitterDelegate {
                 date: glucose.readDate,
                 quantity: quantity,
                 isDisplayOnly: glucose.isDisplayOnly,
+                wasUserEntered: glucose.isDisplayOnly,
```

Comment:
This...I'm not sure of.  It seems legit, but I defer to @ps2  on it. (So if he approves, I approve :) )

Reply:
AFAIK, all calibrations from all supported CGMs are user entered.

Reply:
So, should this just be hard-coded to `true`, then?  Sorry, maybe I just may not understand the purpose of `isDisplayOnly`...

Reply:
No, because of the line above. The source `glucose` could either be from a calibration (`glucose.isDisplayOnly==true`) or a normal CGM reading (`glucose.isDisplayOnly==false`). We just want to mirror that so that a calibration is user entered and a normal CGM reading is not user entered. The source `glucose` in this case did not original track whether it was specifically user entered, but we know that all existing calibrations are, by definition, user entered.  This is correct.

Reply:
My only concern is that I *think*, but I'm not sure, that the "isDisplayOnly" flag doesn't strictly represent the user entered value, but CGM reading that was affected by a user entered value; ie, the user entered value updates the dexcom's internal model.  If the last CGM BG was 300, and you put in a calibration of 200, the dexcom display will now display something like 254.  I think that the 254 is what is reported to Loop (not 100% sure on this, and my attempts to test just now aren't working), so calling it a user entered value might not be completely true.

Reply:
@ps2 What would you like me to do here? I can leave it as-is or, I think the only alternative is to set it to `false`. In theory I could go back and make `wasUserEntered` an optional so that it could be `true`, `false`, or `nil` (meaning "we don't know").

Reply:
I think let's go with setting to true; I tried getting some backfill data with this set, but I can't atm.